FAQ.md

```diff
@@ -58,15 +58,15 @@ In terms of compilers, these are what we currently use for each release:
 
 | Version | OS      | Compiler |
 |---------|---------|----------|
-| JDK8    | Linux   | GCC 4.8 (HotSpot) GCC 7.6 (OpenJ9)                        |
```

Comment:
While it's not a disaster I do personally think this looks easier on the eye lined up, but I wouldn't block it for this change if you disagree :-)

Reply:
No I agree. That looked a lot better in my text editor than here. I'll revert this back what it used to be